src/MCPServer/lib/settings/common.py

```diff
@@ -125,6 +125,16 @@
     }
 }
 
+# These are all the apps that we need so we can use the models in the
+# Dashboard.
+INSTALLED_APPS = (
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'main',
+    'components.administration',
+    'fpr',
+)
```

Comment:
We're just aiming to import `main.models`, but then you need to make sure the app dependencies are available too or it won't work. Ideally (long term) we wouldn't be sharing models/database at all but that's not what we're trying to achieve here.